cdk/src/open-data-platform/root-stack.ts

```diff
@@ -9,22 +9,27 @@ import { FrontendStack } from './frontend/frontend-stack';
 import { AppPlaneStack } from './app-plane/app-plane-stack';
 
 export class OpenDataPlatformStack extends Stack {
-  constructor(scope: Construct, id: string, props?: util.CommonProps) {
+  constructor(scope: Construct, id: string, props: util.CommonProps) {
     super(scope, id, props);
+    const { envType } = props;
 
-    const networkStack = new NetworkStack(scope, util.stackName(util.StackId.Network), {
+    const networkStack = new NetworkStack(scope, util.stackName(util.StackId.Network, envType), {
```

Comment:
Is 'scope' instead of 'this' used intentionally here and in other places? What is the difference?

Reply:
(as discussed) this was unintentional and meant that each of these stacks aren't actually nested stacks.

I've removed the root stack altogether now.